<!DOCTYPE html>
<html>

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <title>CRISPR Gene Editing Technology</title>
  <link rel="stylesheet" type="text/css" href="style1.css"/>
</head>

  <body>
    <div class="history_page">
      <nav>
          <ul>
            <li> <a href="index.html"> Home </a> </li>
            <li> <a href="history.html"> History </a> </li>
            <li> <a href="scientific.html"> Chemistry of CRISPR </a> </li>
            <li> <a href="application.html"> Applications </a> </li>
            <li> <a href="ethics.html"> Ethics </a> </li>
            <li> <a href="sources.html"> Bibliography </a> </li>
          </ul>
      </nav>

      <header>
        <div class="typing">
          <h1> CRISPR Gene Editing Technology </h1>
        </div>
        <h2> History of CRISPR </h2>
      </header>
      <div class="history_content">
        <!-- Slideshow container -->
        <div class="slideshow-container">

          <!-- Full-width images with number and caption text -->
          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/yoshizumi_ishino.png" alt="yoshizumi_ishino" width="498" height="332">
            <div class="text">Yoshizumi Ishino (Japan): First person to see repeating genetic patterns</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/francisco_mojica.png" alt="francisco_mojica" style="width:100%">
            <div class="text">Francisco Mojica (Spain): Finds multiple copies of repeated sequences in RNA</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/gilles_vergnaud.png" alt="gilles_vergnaud" style="width:100%">
            <div class="text">Gilles Vergnaud (France): Showed CRISPR is a defense mechanism</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/philippe_horvath.png" alt="philippe_horvath" style="width:100%">
            <div class="text">Philippe Horvath (France): Showed resistance to phages is due to CRISPR</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/rodolphe_barrangou.png" alt="rodolphe_barrangou" style="width:100%">
            <div class="text">Rodolphe Barrangou (USA): Showed resistance to phages is due to CRISPR</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/sylvian_moineau.png" alt="sylvian_moineau" style="width:100%">
            <div class="text">Sylvian Moineau (Canada): Showed resistance to phages is due to CRISPR</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/john_van_der_oost.png" alt="john_van_der_oost" style="width:100%">
            <div class="text">John van der Oost (Netherlands): Showed the need for CRISPR-RNA (crRNA)</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/luciano_marraffini.png" alt="luciano_marraffini" style="width:100%">
            <div class="text">Luciano Marraffini (USA): Showed CRISPR creates resistance to plasmids</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/emmanuelle_charpentier.png" alt="emmanuelle_charpentier" style="width:100%">
            <div class="text">Emmanuelle Charpentier (France): Showed  that tracrRNA is needed for CRISPR to work</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/jorg_vogel.png" alt="jorg_vogel" style="width:100%">
            <div class="text">Jorg Vogel (Germany): Showed  that tracrRNA is needed for CRISPR to work</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/virginijus_siksnys.png" alt="virginijus_siksnys" style="width:100%">
            <div class="text">Virginijus Šikšnys (Lithuania): Proved that Cas9 is needed to cleave DNA</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/jennifer_doudna.png" alt="jennifer_doudna" style="width:100%">
            <div class="text">Jennifer Doudna (USA): Showed that crRNA and tracrRNA could be in a single strand</div>
          </div>

          <div class="mySlides fade">
            <div class="numbertext"> </div>
            <img src="images/feng_zang.png" alt="feng_zang" style="width:100%">
            <div class="text">Feng Zang (USA): Showed CRISPR-Cas9 could be used in mammalian cells</div>
          </div>

          <!-- Next and previous buttons -->
          <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
          <a class="next" onclick="plusSlides(1)">&#10095;</a>
        </div>

        <!-- The dots/circles -->
        <div style="text-align:center">
          <span class="dot" onclick="currentSlide(1)"></span>
          <span class="dot" onclick="currentSlide(2)"></span>
          <span class="dot" onclick="currentSlide(3)"></span>
          <span class="dot" onclick="currentSlide(4)"></span>
          <span class="dot" onclick="currentSlide(5)"></span>
          <span class="dot" onclick="currentSlide(6)"></span>
          <span class="dot" onclick="currentSlide(7)"></span>
          <span class="dot" onclick="currentSlide(8)"></span>
          <span class="dot" onclick="currentSlide(9)"></span>
          <span class="dot" onclick="currentSlide(10)"></span>
          <span class="dot" onclick="currentSlide(11)"></span>
          <span class="dot" onclick="currentSlide(12)"></span>
          <span class="dot" onclick="currentSlide(13)"></span>
        </div>

        <script>
          var slideIndex = 1;
          showSlides(slideIndex);

          function plusSlides(n) {
          showSlides(slideIndex += n);
          }

          function currentSlide(n) {
          showSlides(slideIndex = n);
          }

          function showSlides(n) {
          var i;
          var slides = document.getElementsByClassName("mySlides");
          var dots = document.getElementsByClassName("dot");
          if (n > slides.length) {slideIndex = 1}
          if (n < 1) {slideIndex = slides.length}
          for (i = 0; i < slides.length; i++) {
              slides[i].style.display = "none";
          }
          for (i = 0; i < dots.length; i++) {
              dots[i].className = dots[i].className.replace(" active", "");
          }
          slides[slideIndex-1].style.display = "block";
          dots[slideIndex-1].className += " active";
          }
        </script>

        <div class="timeline">
          <p> </p>
          <div class="container left">
            <div class="content">
              <h2>1980s</h2>
              <h4>Yoshizumi Ishino (Japan) finds a weird 
                repeating pattern in a genetic sequence.</h4>
              <p>In the 1980’s, researchers at Osaka University, 
                led by Yoshizumi Ishino, were looking at the genetic sequence 
                of the alkaline phosphatase isozyme responsible for conversion 
                aminopeptidase in Escherichia coli when they noticed something particular. 
                In this genetic sequence, the researchers saw that a particular 
                sequence repeated five times in the genetic sequence. This surprised 
                the researchers, and they did not understand why this occurred. They 
                published their findings in the Journal of Bacteriology in 1987.
              </p>
            </div>
          </div>
          <div class="container right">
            <div class="content">
              <h2>1989</h2>
              <h4>Francisco Mojica (Spain) starts his Ph.D. studies and finds 
                multiple copies of the same repeated DNA sequence in a microbe.</h4>
              <p>In 1989, Francisco Mojica began his Ph.D. studies at the University 
                of Alicante by joining a lab working on Haloferax mediterranei, 
                a microbe with extreme salt tolerance. In the first DNA fragment he 
                examined, Mojica found multiple copies of a near-perfect roughly 
                palindromic repeated sequence separated by spacers that did not 
                resemble any known repeats in microbes. These results were published in 1993.
              </p>
            </div>
          </div>
          <div class="container left">
            <div class="content">
              <h2>1995</h2>
              <h4>Francisco Mojica links his previous work to that of Yoshizumi Ishino.</h4>
              <p>In 1995, Mojica published a paper linking his repeats to those found by the 
                team led by Ishino. He realized that the presence of similar structures 
                must signal an important function in prokaryotes (a cellular organism 
                that lacks a membrane-enclosed nucleus).
              </p>
            </div>
          </div>
          <div class="container right">
            <div class="content">
              <h2>2002</h2>
              <h4>Ruud Janssen (Netherlands) publishes the first paper that uses the 
                acronym “CRISPR.”</h4>
              <p>In correspondence between Mojica and Ruud Janssen of Utrecht University in 
                the Netherlands, this series of repeats were given the name “Clustered Regularly 
                Interspaced Short Palindromic Repeats” or “CRISPR.” By 2000, Mojica had 
                found CRISPR loci in 20 different microbes, and in 2002 Ruud Janssen published 
                the first paper using the CRISPR acronym.
              </p>
            </div>
          </div>
          <div class="container left">
            <div class="content">
              <h2>2005</h2>
              <h4>Francisco Mojica publishes a paper showing that CRISPR genes are 
                the encoded instructions for an adaptive immune system.</h4>
              <p>In 2003, Mojica was now focused not on the repeats seen in CRISPR, 
                but on the sequences in the spaces between the repeats “Spacers.” 
                He wanted to understand what these Spacers meant. During the summer,
                Mojica took each sequence and entered them into a computer program 
                that would search DNA databases for matches. His search found a match.
                One of the Spacers that he had entered matched the sequence of a P1 phage 
                that infected many E.coli strains. However, this particular strain that had 
                the Spacer was known to be resistant to P1 phage infections. 
                With this knowledge, Mojica went through 4,500 other Spacers. 
                He found that of the 88 Spacers with similarities to known sequences, 
                almost 67% matched viruses or conjunctive plasmids (a small, extrachromosomal 
                DNA molecule within a cell that is physically separated from chromosomal DNA
                and can replicate independently) related to the microbe carrying the Spacer. 
                Mojica realized that the CRISPR loci must encode the instructions for an adaptive 
                immune system that protected microbes against specific infections. He published 
                his paper on this in 2005.

              </p>
            </div>
          </div>
          <div class="container right">
            <div class="content">
              <h2>2005</h2>
              <h4>Giles Vergnaud (France) and his team publish a paper proposing CRISPR 
                is a defense mechanism to infections and that CRISPR represents the memory of 
                past genetic aggressions.</h4>
              <p>In 1997, Giles Vergnaud led a lab working for the French Ministry of 
                Defense to develop methods of identifying the sources of pathogens based on their DNA. 
                The French Ministry of Defense had 61 Samples of Y. pestis from a plague outbreak in 
                Vietnam in the 1960s. After sequencing the strains, the team found that the strain 
                occasionally differed because new spacers were in front of the CRISPR chains. 
                The team published their paper in 2005, which proposed that CRISPR was a defense 
                mechanism to infections and that CRISPRs represented the memory of past genetic aggressions.
              </p>
            </div>
          </div>
          <div class="container left">
            <div class="content">
              <h2>2005</h2>
              <h4>Philippe Horvath (France), Rodolphe Barrangou (USA), and Sylvian Moineau (Canada)
                discovered that bacteria develop resistance to phages by having additional sequences 
                added to the CRISPR locations. They also saw that Cas9 was required to cut nucleic acids.</h4>
              <p>Also, in 2005, Philippe Horvath at Rhodia Foods worked with colleagues, including 
                Rodolphe Barrangou at Danisco USA and Sylvian Moineau at Université Laval in Canada. 
                The team used a well-characterized phage-sensitive strain of bacteria (S. thermophilus)
                and two bacteriophages to conduct their experiments. They found that instead of having 
                classical resistance mutations (such as in a cell-surface receptor required for phage
                entry), the resistant strains had acquired phage-derived sequences in their CRISPR 
                locations. The team also studied the role of two Cas genes, Cas7 and Cas9. They found 
                that bacteria need Cas7 in order to gain resistance, but those carrying a phage-derived 
                spacer did not need the gene to remain resistant, suggesting that Cas7 was involved in
                generating new spacers and repeats, but not in immunity itself. In contrast, Cas9, 
                whose product presumably cut nucleic acids, was an active component of the bacterial 
                immune system.

              </p>
            </div>
          </div>
          <div class="container right">
            <div class="content">
              <h2>2005</h2>
              <h4>John van der Oost (Netherlands) starts to study CRISPR. He and his team start to understand 
                how the Cas proteins work, and that long CRISPR RNA (“crRNA”) is responsible for the 
                CRISPR based resistance.</h4>
              <p>Also, in 2005, John van der Oost at the Free University of Amsterdam started to study CRISPR. 
                He and colleagues inserted an E. coli CRISPR system into another E. coli system that lacked
                its own system. This allowed the researchers to understand how the five Cas proteins, 
                termed “Cascade,” worked. By removing each component of the cascade in turn, the researchers 
                showed that a cascade is required to cleave long recurs “RNA” transcribed from the CRISPR 
                locus into 61-nucleotide long CRISPR RNAs (“crRNA”). To prove that the crRNA sequences are 
                responsible for the CRISPR-based resistance, they set out to create the first artificial 
                CRISPR arrays by programming CRISPR to target four essential genes in lambda phage. 
                As predicted, the strains carrying new CRISPR sequences showed resistance to phage lambda.
              </p>
            </div>
          </div>
          <div class="container left">
            <div class="content">
              <h2>Mid-2000s</h2>
              <h4>Luciano Marraffini (USA) showed that the CRISPR system also blocked plasmids 
                just like it blocked phage infections. He also showed with Erik Sontheimer (USA) 
                that the infection needed the same genetic sequence that CRISPR already had in 
                its locations to block the infection.</h4>
              <p>Luciano Marraffini was completing his Ph.D. at the University of Chicago in the 
                mid 2000’swhen he learned about CRISPR from a colleague, Malcolm Casadaban, 
                who is a leading authority on phages. He started working on CRISPR as he 
                completed his graduate studies by exploring whether the Staphylococcus CRISPR 
                system could block plasmids. Upon completion of his studies, he moved to 
                Northwestern University to work with Erik Sontheimer. There, Maraffini noticed 
                that a strain of Staphylococcus epidermidis had a spacer that matched a region 
                of a gene coded on plasmids from antibiotic-resistant Staphylococcus aureus. He 
                showed that these plasmids could not be transferred to the S. epidermidis, but 
                by disrupting either the genetic sequence as encoded in the spacer region in the
                plasmid or the matching spacer sequence in the CRISPR locus of the genome, 
                resistance was abolished. This showed that CRISPR blocked plasmids, just as 
                it blocked viruses.
              </p>
              <p>
                Marraffini and Sontheimer then continued their experiments by splicing an “intron” 
                (a segment of a DNA molecule that does not code for proteins & interrupts the 
                sequence of a gene) into the middle of the gene fragment they had identified 
                being targeted by CRISPR. If CRISPR targeted mRNA, this change would have no impact. 
                If CRISPR targeted DNA, the resistance would no longer exist as the sequence in CRISPR,
                and the DNA would no longer match. They found the resistance no longer existed. Thus,
                CRISPR targets DNA.

              </p>
            </div>
          </div>
          <div class="container right">
            <div class="content">
              <h2>2007</h2>
              <h4>2007 Sylvian Moineau with colleagues used a strain of S. thermophilus to
                see how the process of cutting DNA worked.</h4>
              <p>Following the publication of his study in 2007, Sylvian Moineau continued to 
                work with Danisco to understand the mechanism by which CRISPR cleaves DNA. 
                Normally, CRISPR is so efficient at cleaving DNA that researchers cannot see 
                the process in action. However, Moineau and his colleagues got lucky while 
                studying plasmid interference in S. thermophilus. They found a handful of 
                bacterial strains in which CRISPR conferred only partial protection against
                plasmids. In one such inefficient strain, they could see linearized plasmids 
                persisting inside the cells. The process of plasmid interference had been slowed 
                down enough for the team to see the direct products of CRISPR’s actions. 
                This strain actually allowed the researchers to dissect the process of cutting 
                itself. Just like their previous studies, the team showed that cutting DNA was 
                dependent on the Cas9 gene and that the cutting took place three nucleotides 
                upstream of the spacer sequence (or proto-spacer adjacent motif “PAM”). 

              </p>
            </div>
          </div>
          <div class="container left">
            <div class="content">
              <h2>2007</h2>
              <h4>Emmanuelle Charpentier (France) and Jorg Vogel 
                (Germany) meet at a meeting of the “RNA Society.”
                Working together, they used the latest genetic sequencing technology
                to show that trans-activating CRISPR RNA or “tracrRNA” was used in the 
                cleavage of RNA.</h4>
              <p>Emmanuelle Charpentier and Jorg Vogel, who met at the 2007 meeting of 
                RNA society in Madison, WI, teamed up to try and identify microbial RNA. 
                Using recently developed “next-generation sequencing” technology, Vogel 
                realized they could produce comprehensive catalogs of any microbial 
                transcripted RNA. Thus Charpentier and Vogel used this approach on S. pyogenes.
                They found that the third most common RNA class was from a sequence immediately 
                adjacent to the CRISPR locus and had 25 bases of near-perfect complementary to 
                the CRISPR repeats. The complementarity suggested this RNA sequence of 
                “trans-activating CRISPR RNA” (tracrRNA) was used in the cleavage of RNA. 
                Genetic deletion experiments confirmed this notion, showing tracrRNA was 
                essential for processing crRNA and thus CRISPR function. Later studies would
                show that tracrRNA was also essential for Cas9 to cleave DNA.

              </p>
            </div>
          </div>
          <div class="container right">
            <div class="content">
              <h2>2007</h2>
              <h4>Virginijus Šikšnys (Lithuania) starts working on CRISPR. He shows that 
                transferring a CRISPR locus from one bacteria to another can 
                induce resistance. In addition, he shows only Cas9 protein is the only 
                protein needed to cause interference.</h4>
              <p>Virginijus Šikšnys, working at the Institute of
                Applied Enzymology in Lithuania, became interested 
                in bacterial barriers to foreign DNA after reading Horvath, Borrango, 
                and Moineau’s 2007 paper. So he and his collaborators set out to see if 
                the CRISPR system from one bacteria (S. thermophilus) could be reconstituted 
                in another (E. coli). They found that transferring the entire CRISPR locus 
                was sufficient to cause targeted interference against plasmid and 
                bacteriophage DNA. They also proved that the Cas9 protein is the only
                protein required for interference.
              </p>
            </div>
          </div>
          <div class="container left">
            <div class="content">
              <h2>2011</h2>
              <h4> Emmanuelle Charpentier gives a lecture in Puerto Rico and meets Jennifer Doudna
                (USA). They work together to understand the function of CRISPR and Cas9. 
                They show that both crRNA and TracrRNA are needed. In addition, 
                they show the 2 RNA groups could function when fused into a single guide RNA, 
                “SgRNA.”</h4>
              <p>In March 2011, Charpentier gave a lecture on tracrRNA at the American Society 
                for Microbiology meeting in Puerto Rico. Here she met Jennifer Doudna, 
                a world-renowned structural biologist and RNA expert at the University of 
                California, Berkeley. Doudna had been using crystallography and cryo-electron
                microscopy to understand the structures of the Cascade CRISPR system, 
                the more complex type of CRISPR system used by microbes such as E. coli. 
                The two scientists decided to join forces. They used recombinant Cas9 
                (from S. pyogenes expressed in E. coli) and crRNA and tracrRNA that had been
                transcribed in vitro. Like Šikšnys, the two scientists showed that Cas9 could 
                cut purified DNA in vitro and that it could be programmed with custom-designed 
                crRNAs so that they could cut opposite strands of DNA, and that both crRNA and 
                tracrRNA were needed for Cas9 to function. In addition, they showed that the 
                two RNA groups could function in vitro when they were fused into a single-guide 
                RNA “sgRNA.” The idea of sgRNA would become widely used in genome editing.

              </p>
            </div>
          </div>
          <div class="container right">
            <div class="content">
              <h2>2011</h2>
              <h4>Feng Zang (USA) creates a version of CRISPR-Cas9 that can be used in mammalian cells.</h4>
              <p>In the late 1980s, scientists devised a way to change mammalian genomes’ 
                living cells, which transformed biomedical research. However, till now, the
                process for transforming the genomes was very inefficient. With the discovery 
                of CRISPR and Cas9, researchers believed they had a powerful tool, but they 
                did not know if it would work. As late as September 2012, scientists were 
                skeptical that they could use these tools to edit genomes. Feng Zang, who 
                got his Ph.D. from Stanford, heard a talk on CRISPR from Michael Gilmore, a 
                Harvard Microbiologist, in February 2011. Zang was captivated by this process. 
                After the talk, he flew to a scientific meeting in Miami but remained holed up 
                in his hotel room digesting the entire CRISPR literature. When he returned, he 
                set out to create a version of S. thermophilus Cas9 for use in human cells. 
                He found that his version was unevenly distributed within the nucleus. 
                Because it clumped in the nucleus, Zang then started testing other microbe 
                systems and found that S. pyogenes Cas9 was much better distributed. He found 
                that even though mammalian cells lacked microbial RNA, they could still process 
                crRNA, although differently than how bacteria did. He tested various forms of 
                tracrRNA to identify one that was stable in human cells. By mid-2012, he had
                developed a stable Cas9 CRISPR system for use in human cells. After reading the
                results of Charpentier and Doudna’s work, Zang tested a two-component system with 
                sgRNA. However, the simplified system did not work efficiently. Zang found that
                if he used a full fusion of the crRNA and tracrRNA, it worked efficiently. Zang’s
                paper on mammalian genome editing was published in January 2013, and his non-profit 
                Addgene would receive over 25,000 requests for the reagents needed to use the
                techniques he described.

              </p>
            </div>
          </div>
          <div class="container left">
            <div class="content">
              <h2>2012</h2>
              <h4>Feng Zang has created a stable version of the CRISPR-Cas9 system, and that it worked 
                with the sgRNA as proposed by Charpentier and Doudna.</h4>
              <p>About a month after Zang submitted his paper for publishing, George Church, a senior
                Harvard professor with deep expertise in genomics and synthetic biology, submitted a 
                paper on genome editing in human cells. He had used the full-length fusion techniques for 
                sgRNA to show that he could cut and join DNA in mammalian cells. His paper appeared back to 
                back with Zang’s paper. These two papers brought great interest in the field, and scientists
                around the world have been working on using them to edit genomes.
              </p>
            </div>
          </div>
          <p> </p>
        </div>
      </div>

      <div class="citations_box">
        <h4>History Bibliography</h4>
        <ol>
          <li>Han, W.; She, Q. CRISPR History: Discovery, Characterization, and Prosperity. https://www.sciencedirect.com/science/article/pii/S1877117317301382 (accessed May 17, 2021).</li>
          <li>Ishino, Y.; Shinagawa, H.; Makino, K.; Amemura, M.; Nakata, A. Nucleotide Sequence of the iap Gene, Responsible for Alkaline Phosphatase Isozyme Conversion in Escherichia coli, and Identification of the Gene Product 1987, 169 (12), 5429–5433.</li>
          <li>Lander, E. S. The Heroes of CRISPR. https://www.broadinstitute.org/files/news/pdfs/PIIS0092867415017055.pdf (accessed May 17, 2021).</li>
        </ol>
      </div>

      <footer>
        <h4> This website was created by Arshia Agrawal and Ariha Mehta. </h4>
        <h4> Chemistry Olympics 2021 </h4>
      </footer>
    </div>
  </body>
</html>
